.gitignore

```diff
@@ -151,3 +151,5 @@ localdev-*
 # OS generated
 .DS_Store
 .DS_Store?
+/docs-source/source/auto_examples
+/docs-source/source/auto_examples
```

Comment:
You should not commit .`gitignore` in this PR. 
auto_examples will be used when we release documentation, so we should not ignore it in the main branch, but you should ignore it in your PR :)

Reply:
Committed by accident. Should be corrected now.